File: Python/FindPeakElement.py
# A peak element is an element that is greater than its neighbors.

# Given an input array where num[i] ≠ num[i+1], find a peak element and return its index.

# The array may contain multiple peaks, in that case return the index to any one of the peaks is fine.

class Solution(object):
    def findPeakElement(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        biggest = max(nums);
        result = 0;
        while (nums[result] != biggest):
            result += 1;
        return result;

File: C/FindPeakElement.c
/* A peak element is an element that is greater than its neighbors.

  Given an input array where num[i] ≠ num[i+1], find a peak element and return its index.

  The array may contain multiple peaks, in that case return the index to any one of the peaks is fine. */

  int findPeakElement(int* nums, int numsSize) {
    int biggest = nums[0];
    int index = 0;
    int result = 0;
    while (index < numsSize) {
        if (nums[index] >= biggest) {
            biggest = nums[index];
            result = index;
        }
        index += 1;
    }
    return result;

}
